spec/requests/restaurants_spec.rb

```diff
@@ -2,6 +2,42 @@
 
 RSpec.describe "Restaurants", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of restaurants" do
+      Restaurant.create(
+        # name: 'Grubbers Bar&Grill',
```

Comment:
Do you have plans for name?

Reply:
We have it fixed on the main branch, the migration was created after we thought of name